.github/workflows/ci.yml

```diff
@@ -44,7 +44,7 @@ jobs:
       # Used to host cibuildwheel
       - uses: actions/setup-python@v5
         with:
-          python-version: "3.12"
+          python-version: "3.13"
```

Comment:
shouldn't this be using version from strategy instead of hard coding?

Reply:
yeah! good catch. I changed to use the python version from the matrix

Reply:
Actually, I think is fine to use 3.13 because it's only been used for tooling, not for the wheels. I reset it then.